articles/quickstart/spa/angular-next/02-calling-an-api.md

```diff
@@ -0,0 +1,151 @@
+:::
+
+## Provide the HTTP Interceptor
+
+To install and configure the HTTP interceptor, perform the following steps:
+
+* Import the `AuthHttpInterceptor` type from the Auth0 Angular SDK
+* Import `HttpClientModule` and `HTTP_INTERCEPTORS` from `@angular/common/http`
+* Register `AuthHttpInterceptor` in the `providers` section of your application module
+* Add configuration to specify audience, scope, and which requests should have an `Authorization` header attached automatically
+
+The following is an example of an Angular module (based upon the default implementation when you create a new app using `ng new`) that supports `AuthHttpInterceptor`, configured to call the Auth0 Management API with the ability to read the current user profile:
+
+To begin, open your `app.module.ts` file and add the necessary imports at the top:
+
+```javascript
+// Import the injector module and the HTTP client module from Angular
+import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
+
+// Import the HTTP interceptor from the Auth0 Angular SDK
+import { AuthHttpInterceptor } from '@auth0/auth0-angular';
+```
+
+Next, import `HttpClientModule` into the `imports` array of `AppModule`:
+
+```javascript
+imports: [
+  HttpClientModule,
+],
+```
+
+Now add the Auth0 HTTP interceptor to the `providers` array of `AppModule`, like so:
+
+```javascript
+providers: [
+  { provide: HTTP_INTERCEPTORS, useClass: AuthHttpInterceptor, multi: true },
+],
+```
+
+Finally, modify the configuration given to `AuthModule.forRoot()` to specify the `audience` and `scope` values required by the API you want to call, as well as the API routes that should be intercepted by `AuthHttpInterceptor`.
+
+In this case, the audience and scope for the Auth0 Management API are given, which allows your app to retrieve information about the current user.
+
+```javascript
+AuthModule.forRoot({
+  // The domain and clientId were configured in the previous chapter
+  domain: "${account.namespace}",
+  clientId: "${account.clientId}",
+
+  // Request this audience at user authentication time
+  audience: "https://${account.namespace}/api/v2/",
+
+  // Request this scope at user authentication time
+  scope: "read:current_user",
+
+  // Specify configuration for the interceptor              
+  httpInterceptor: {
+    allowedList: [
+      {
+        // Match any request that starts 'https://${account.namespace}/api/v2/' (note the asterisk)
+        uri: 'https://${account.namespace}/api/v2/*'
+        tokenOptions: {
+          // The attached token should target this audience
+          audience: 'https://${account.namespace}/api/v2/',
+
+          // The attached token should have these scopes
+          scope: 'read:current_user'
+        }
+      }
+    ]
+  }
+})
+```
+
+Please [refer to the docs](https://github.com/auth0/auth0-angular#configure-authhttpinterceptor-to-attach-access-tokens) for more information on the available options for the HTTP interceptor.
```

Comment:
We can link to the library docs here when they are published. Right now, I have no idea what that link would be.